java/src/test/java/com/xxAMIDOxx/xxSTACKSxx/ActuatorTest.java

```diff
@@ -10,8 +10,12 @@
 
 import java.util.Map;
 
+import static com.xxAMIDOxx.xxSTACKSxx.api.v1.menu.utilities.HelperMethods.getBaseURL;
```

Comment:
This is generic.
I would put in a higher up package. e.g. `com.xxAMIDOxx.xxSTACKSxx.util`